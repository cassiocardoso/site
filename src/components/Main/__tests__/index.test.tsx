import { render, screen } from "@testing-library/react";

import { Main } from "../index";

describe("components/Main", () => {
  it("should render properly", () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole("heading", { name: /main component/i }),
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
