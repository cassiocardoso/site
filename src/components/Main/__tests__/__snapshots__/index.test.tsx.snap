// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Main should render properly 1`] = `
.c0 {
  background: #fafef0;
  text-align: center;
}

<main
  class="c0"
>
  <h1>
    main component
  </h1>
</main>
`;
